src/Command/CodeStudio/CodeStudioCommandTrait.php

```diff
@@ -207,11 +207,16 @@ protected function determineGitLabProject(ApplicationResponse $cloud_application
     }
     // Prompt to create project.
     else {
-      $this->io->writeln([
+      $error_message = [
         "",
-        "Could not find any existing Code Studio project for Acquia Cloud Platform application <comment>{$cloud_application->name}</comment>.",
-        "Searched for UUID <comment>{$cloud_application->uuid}</comment> in project descriptions.",
-      ]);
+        "Could not find any existing Code Studio project for Acquia Cloud Platform application {$cloud_application->name}.",
+        "Searched for UUID {$cloud_application->uuid} in project descriptions.",
+      ];
+      if (!$create_new_cs_project) {
+        $this->io->error($error_message);
```

Comment:
Please add a test for the presence of this error message so that the escaped mutant is squashed.